doc/api/events.md

```diff
@@ -2071,7 +2068,29 @@ added: v14.5.0
 Node.js-specific extension to the `EventTarget` class that returns the number
 of event listeners registered for the `type`.
 
-#### `nodeEventTarget.off(type, listener)`
+#### `nodeEventTarget.setMaxListeners(n)`
+
+<!-- YAML
+added: v14.5.0
+-->
+
+* `n` {number}
+
+Node.js-specific extension to the `EventTarget` class that sets the number
+of max event listeners as `n`.
+
+#### `nodeEventTarget.getMaxListeners()`
+
+<!-- YAML
+added: v14.5.0
+-->
+
+* Returns: {number}
+
+Node.js-specific extension to the `EventTarget` class that returns the number
+of max event listeners.
```

Comment:
We would need to update this list as well: https://github.com/nodejs/node/blob/016abb6c559fded98492d8df7d1c208f5e5c6c8f/doc/api/events.md#L1646-L1650

Reply:
Updated list of unsupported function in `NodeEventTarget`. Thank you for comment.